tests/integration/features/bootstrap/SharingContext.php

```diff
@@ -596,9 +596,15 @@ public function shareXIsReturnedWith(int $number, TableNode $body) {
 			'item_source' => 'A_NUMBER',
 			'file_source' => 'A_NUMBER',
 			'file_parent' => 'A_NUMBER',
-			'mail_send' => '0'
+			'mail_send' => '0',
 		];
-		$expectedFields = array_merge($defaultExpectedFields, $body->getRowsHash());
+
+		$fields = $body->getRowsHash();
+		if (isset($fields['share_type']) && ($fields['share_type'] === '10' || $fields['share_type'] === '11')) {
```

Comment:
All expected values on `*.feature` files is with share_type are `0`. Now we haven't integration tests covering scenario with share_type 10 or 11.

Reply:
There is a fallback to 10 in line 589 of this file. So basically each test which does not specify a share_type is 10 and therefor matching this case